pages/Roles-and-Responsibilities.md

```diff
@@ -5,23 +5,17 @@
 
-## 18F Procurement Project Manager:
+## TTS Product Lead:
+The TTS Product Lead is the interface between the requiring organization (GSA-TTS) and contracting organization (GSA FAS AAS/NCR). The Product Lead coordinates technical aspects of the contract with the COR/ACOR and CO, assists with contract administration, ensures client acceptance of services, and reviews invoices for payment.  The Product Lead is also responsible for coordination between the contractor, the Product Owner, as well as the CO and COR/ACOR.
 
+## TTS Technical Lead:
 
-The Procurement Project Manager is the interface between the requiring organization (GSA 18F) and contracting organization (GSA FAS AAS/NCR). The Procurement PM coordinates technical aspects of the contract with the COR/ACOR and CO, assists with contract administration, ensures client acceptance of services, and reviews invoices for payment.
-
-## 18F Product Lead:
-
-The 18F Product Lead is responsible for coordination between the contractor, the Product Owner, as well as the CO and COR/ACOR.
-
-## 18F Technical Lead:
-
-The 18F technical lead is responsible for coordination between the contractor’s development team, the Product Owner and the key interagency stakeholders.
+The TTS technical lead is responsible for coordination between the contractor’s development team, the Product Owner and the key interagency stakeholders.
 
 ## Product Owner (Partner Agency):
 
-The Product Owner is the project’s key stakeholder and is from the e-Permitting Team. They are responsible for having a vision of what he or she wishes to build, and convey the vision to the 18F Product Lead and the contractor’s scrum or development team. The Product Owner does this in part through the product backlog, which will be a prioritized features list for the product. The Product Owner is responsible for advocating on behalf of the agency’s needs and responsible for decision-making during sprint planning and implementation. The Product Owner will work with the COR/ACOR to inspect vendor work.
+The Product Owner is the project’s key stakeholder and is from the e-Permitting Team. They are responsible for having a vision of what he or she wishes to build, and convey the vision to the TTS Product Lead and the contractor’s scrum or development team. The Product Owner does this in part through the product backlog, which will be a prioritized features list for the product. The Product Owner is responsible for advocating on behalf of the agency’s needs and responsible for decision-making during sprint planning and implementation. The Product Owner will work with the COR/ACOR to inspect vendor work.
```

Comment:
Should this be something about the "partner agency team" rather than "e-Permitting Team"?